src/priyankaKamble/assignment_21/ReturnDigitString.java

```diff
@@ -0,0 +1,32 @@
+package priyankaKamble.assignment_21;
+
+import java.util.Scanner;
+
+/*Create a method which will written the string with all the digits from the given string 
+Input : "Tec1hn2o3C9r6e8d1i9ts"
+Output : 12396819*/
+public class ReturnDigitString {
+	
+	String result ="";
```

Comment:
if this variable is common for other methods then we should write it at the instance level.